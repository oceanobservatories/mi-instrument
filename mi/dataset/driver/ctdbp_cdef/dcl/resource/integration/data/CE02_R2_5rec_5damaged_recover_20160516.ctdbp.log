2016/05/16 00:00:08.819 [ctdbp:DLOGP3]:Instrument Started [Power On]
2016/05/16 21:00:24.160 # 11.9168,  3.62176,    6.847, 16 May 2016 21:00:29
2016/05/16 21:00:34.120 # 3.62154,    6.839, 16 May 2016 21:00:39
2016/05/16 21:00:44.127 # .9154,  3.62174,    6.974, 16 May 2016 21:00:49
2016/05/16 21:00:54.120 # 11.9120,  6.963, 16 May 2016 21:00:59
2016/05/16 21:01:04.119 # 11.9139,  3.62200,    16 May 2016 21:01:09
2016/05/16 21:01:14.120 # 11.9124,  .62187,    6.894, 16 May 2016 21:01:19
2016/05/16 21:01:24.119 # 11.9147,  3.62175,    7.010, 16 May 2016 21:01:29
2016/05/16 21:01:34.174 # 11.9127,  3.62200,    6.847, 16 May 2016 21:01:39
2016/05/16 21:01:44.119 # 11.9124,  3.62185,    6.776, 16 May 2016 21:01:49
2016/05/16 21:01:54.120 # 11.9086,  3.62220,    6.675, 16 May 2016 21:01:59
2016/05/16 21:03:19.468 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
