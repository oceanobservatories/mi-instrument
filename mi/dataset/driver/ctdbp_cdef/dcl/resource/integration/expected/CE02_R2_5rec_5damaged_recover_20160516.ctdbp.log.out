
 subsite  | node  | sensor       | bin        | method    | time                  | deployment | internal_timestamp | port_timestamp        | preferred_timestamp | quality_flag | conductivity | date_time_string | dcl_controller_timestamp | pressure | temp
----------+-------+--------------+------------+-----------+-----------------------+------------+--------------------+-----------------------+---------------------+--------------+--------------+------------------+--------------------------+----------+---------
 CE02SHSM | RID27 | 03-CTDBPC000 | 3672345600 | recovered | 3672421224.1599998474 |          2 |         3672421229 | 3672421224.1599998474 |      port_timestamp |           ok |      3.62176 |             null |                     null |    6.847 | 11.9168
 CE02SHSM | RID27 | 03-CTDBPC000 | 3672345600 | recovered |  3672421284.118999958 |          2 |         3672421289 |  3672421284.118999958 |      port_timestamp |           ok |      3.62175 |             null |                     null |     7.01 | 11.9147
 CE02SHSM | RID27 | 03-CTDBPC000 | 3672345600 | recovered | 3672421294.1739997864 |          2 |         3672421299 | 3672421294.1739997864 |      port_timestamp |           ok |        3.622 |             null |                     null |    6.847 | 11.9127
 CE02SHSM | RID27 | 03-CTDBPC000 | 3672345600 | recovered |  3672421304.118999958 |          2 |         3672421309 |  3672421304.118999958 |      port_timestamp |           ok |      3.62185 |             null |                     null |    6.776 | 11.9124
 CE02SHSM | RID27 | 03-CTDBPC000 | 3672345600 | recovered | 3672421314.1199998856 |          2 |         3672421319 | 3672421314.1199998856 |      port_timestamp |           ok |       3.6222 |             null |                     null |    6.675 | 11.9086

(5 rows)
